docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/seedwork-domain-model-base-classes-interfaces.md

```diff
@@ -88,7 +88,7 @@ public abstract class Entity
     public static bool operator ==(Entity left, Entity right)
     {
         if (Object.Equals(left, null))
-            return (Object.Equals(right, null)) ? true : false;
+            return (Object.Equals(right, null));
```

Comment:
I didn't see the redundant parenthesis… 🙈
should I change it some day with other changes in a new PR?

Reply:
which redundant parenthesis?

Reply:
`return Object.Equals(right, null);`

should be enough.

Reply:
I didn't notice the redundant parentheses either, @MichaelWhi and @MichaelWhi, I apologize. Would you like to open a new PR to remove them, @MichaelWhi?